jpa/tck-dist/src/main/resources/eftl-1.1.adoc

```diff
@@ -0,0 +1,23 @@
+= Eclipse Foundation Technology Compatibility Kit License - v1.1
+
+The accompanying Technology Compatibility Kit is "Copyright (c) [$date-of-document], Eclipse Foundation AISBL and its licensors."
```

Comment:
Is the value $date-of-document being replaced somewhere ?

Reply:
As per my current understanding License file should be bundled without any modification.
 @scottmarlow - Any inputs on the comment?

Reply:
So is it for sure the `[$date-of-document]` is meant to be there as it is?

Reply:
@lukasj Feel free to approve then :)